extensions/interactions/MusicNotesInput/directives/MusicNotesInput.js

```diff
@@ -314,7 +325,7 @@ oppia.directive('oppiaInteractiveMusicNotesInput', [
                       tolerance: 'intersect',
                       revert: function() {
                         var draggableOptions = $(this);
-                        // If note is out of droppable or off staff, remove it.
+                        // If note is out of droppable or off staff, remove it
```

Comment:
Keep the periods at the end of comments. Use a second line for the comment if necessary.

Reply:
Done